go/pio/host/chip/chip.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2016 Marc-Antoine Ruel. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package chip
+
+import (
+	"github.com/maruel/dlibox/go/pio/drivers"
+	"github.com/maruel/dlibox/go/pio/host/allwinner"
+	"github.com/maruel/dlibox/go/pio/host/headers"
+	"github.com/maruel/dlibox/go/pio/host/internal"
+	"github.com/maruel/dlibox/go/pio/host/sysfs"
+	"github.com/maruel/dlibox/go/pio/protocols/analog"
+	"github.com/maruel/dlibox/go/pio/protocols/gpio"
+	"github.com/maruel/dlibox/go/pio/protocols/pins"
+)
+
+// Version is the board version. Only reports as 1 for now.
```

Comment:
I wonder about the value of this one and the one in pine64. maybe a set of constants with the different models would be more useful, for pine64 there's is currently 3 different models. From RPI, there's a lot.